students/JeroenSweerts/Lesson3/mailroom.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+donors = ['papa', 'mama', 'bompa', 'bobonne', 'onbekende']
+donations = [[100, 5, 15], [12, 200, 2, 66], [1000], [500, 500], [1000000]]
+
+def thankyou():
+    '''If the user types ‘list’, show them a list of the donor names and re-prompt
+    If the user types a name not in the list, add that name to the data structure
+    and use it.
+    If the user types a name in the list, use it.
+    Once a name has been selected, prompt for a donation amount.
+    Turn the amount into a number – it is OK at this point for the program to
+    crash if someone types a bogus amount.
+    Once an amount has been given, add that amount to the donation history of
+    the selected user.
+    Finally, use string formatting to compose an email thanking the donor for their
+    generous donation. Print the email to the terminal and return to the original
+    prompt.'''
+    name = input('Please enter a name or type "list" to see a list of names > ')
+    while name == 'list':
+        print(donors)
+        name = input('Please enter a name or type "list" to see a list of names > ')
+    if not name in donors:
+        donors.append(name)
+        donations.append([])
+    donation = input('Please enter a donation amount > ')
```

Comment:
Remember to convert the user input from a string to the correct type here. The program currently crashes at line 51 below with the following error message:

```
TypeError: unsupported operand type(s) for +: 'int' and 'str'
```